src/main/scala/org/apache/spark/qbeast/config.scala

```diff
@@ -26,6 +26,18 @@ package object config {
       .intConf
       .createWithDefault(2)
 
+  private[config] val min_file_size_compaction: ConfigEntry[Int] =
```

Comment:
Please use camel style for member names.

Reply:
Solved on 93af690. Changed all the other member names as well.